cmd/runtimetest/main.go

```diff
@@ -576,6 +576,25 @@ func validateMaskedPaths(spec *rspec.Spec) error {
 	return nil
 }
 
+func validateSeccomp(spec *rspec.Spec) error {
+	if spec.Linux == nil || spec.Linux.Seccomp == nil {
+		return nil
+	}
+	for _, sys := range spec.Linux.Seccomp.Syscalls {
+		if sys.Action == "SCMP_ACT_ERRON" {
+			for _, name := range sys.Names {
+				if name == "getcwd" {
```

Comment:
Can we write `t.Skip`s for other names and actions?  I don't want to give users the false impression that we're testing them by silently ignoring other values.

Reply:
> Can we write `t.Skip`s…

This is probably blocking on passing the [TAP instance][1] down into the validators, which I do in #308.  I don't see a clean way to handle this case before #308 lands; if maintainers want a stub in the meantime that's fine, and I don't really care if/how this gets handled in the stub.

[1]: https://github.com/opencontainers/runtime-tools/blob/c3755c1175a1baad7230dfb03751e3a4b7182f32/cmd/runtimetest/main.go#L889

Reply:
Does that mean that all the action and name combinations will be exported as skipped information? Maybe I misunderstood.

Reply:
> Does that mean that all the action and name combinations will be exported as skipped information?

Yes.  That would make it easy for folks writing tests in `validation` to see what `runtimetest` was ignoring.